src/BiometryService/BiometryService.Tests.cs

```diff
@@ -0,0 +1,54 @@
+﻿#if NETSTANDARD2_0
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Logging;
+
+namespace BiometryService;
+
+/// <summary>
+/// Implementation of the <see cref="IBiometryService" /> for Tests.
+/// </summary>
+public sealed partial class BiometryService : IBiometryService
```

Comment:
This isn't a real implementation so I think it would be better to use a name that reflects that.

Reply:
Well in technical terms it actually is. Isn't it cleaner that way?

Reply:
It's a test implementation. It's not encrypting anything. It's not using anything related to biometry.

Reply:
I need to have access to partial class `BiometryService` code to replicate the original behavior, this is a problem if we want a different name.

Reply:
Do you actually need to reuse the code from the BiometryService partial class?

Reply:
Yes and no. The only reason I want to use it, is to better replicate the actual platforms implementations and to not duplicate any code.

Reply:
Problem solved.